config/template_values.yml

```diff
@@ -28,14 +28,22 @@ Creativity Round 2:
              "Tuesday, July 1 - Sunday, July 6, 2025: Firefly!",
              "Friday, July 18, 2025: Firefly documentation due",
              "Friday, Aug 1, 2025: Second check cut and sent"]
-Legacy:
+Legacy Round 1:
+  contract_deadline: "Sunday, April 27, 2025"
+  schedule: ["Sunday, April 6, 2025: Submissions Due",
+             "Sunday, April 27, 2025: Signed Contracts Due",
+             "Sunday, May 11, 2025: First check cut and sent",
+             "Tuesday, July 1 - Sunday, July 6, 2025: Firefly!",
+             "Friday, July 18, 2025: Firefly documentation due",
+             "Friday, Aug 1, 2025: Second check cut and sent"]
+Legacy Round 2:
   contract_deadline: "Sunday, May 11, 2025"
   schedule: ["Sunday, April 27, 2025: Submissions Due",
              "Sunday, May 11, 2025: Signed Contracts Due",
              "Sunday, May 25, 2025: Check cut and sent",
              "Tuesday, July 1 - Sunday, July 6, 2025: Firefly!"]
```

Comment:
Hm, there's no schedule entries for documentation due and second check cut and sent for this part?

My best guess is this is because we actually just give Legacy Grant checks upfront right? So, that would mean the documentation/second check round doesn't apply for these grants. we can check with each other whether this is how it's been in the past, I think it has but my memory is fallible